sdk/messaging/azeventhubs/internal/blob/ci.yml

```diff
@@ -0,0 +1,28 @@
+trigger:
```

Comment:
Presumably this file doesn't need to be here.

Reply:
Done. Also, removed all the .md files.